packages/shared/src/modules/reviews/libs/types/review-get-all-item-response-dto.type.ts

```diff
@@ -0,0 +1,15 @@
+type ReviewGetAllItemResponseDto = {
+	content: string;
+	id: number;
+	likesCount: number;
+	poiId: null | number;
+	routeId: null | number;
+	user: {
+		avatar: null | { url?: null | string };
+		firstName: string;
+		id: number;
+		lastName: string;
+	};
```

Comment:
Can we describe like this?
```suggestion
		user: Pick<
		UserAuthResponseDto,
		"avatarUrl" | "firstName" | "id" | "lastName"
	>;
```

Reply:
https://github.com/BinaryStudioAcademy/bsa-2025-smartscapes/pull/461#discussion_r2292225621